backend/medtagger/api/scans/service_rest.py

```diff
@@ -103,10 +105,15 @@ class Label(Resource):
     def post(scan_id: ScanID) -> Any:
         """Save new label for given scan."""
         payload = request.json
-        selections = payload['elements']
+        elements = payload['elements']
+        try:
+            validate(elements, elements_schema)
+        except ValidationError:
+            raise InvalidArgumentsException('Wrong elements schema.')
```

Comment:
Can we try to be more specific here? I believe that this exception has more details with exact information what is wrong. It may be helpful for users to fix their issues with payload.

- more about this exception: https://python-jsonschema.readthedocs.io/en/latest/errors/#jsonschema.exceptions.ValidationError
- this function may be helpful: https://python-jsonschema.readthedocs.io/en/latest/errors/#jsonschema.exceptions.best_match

BTW. This could be a perfect case for a test ;)